http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -769,10 +776,13 @@ class DataprocInterpreter[F[_]: Parallel](
       _ <- logger.debug(ctx.loggingCtx)(s"Resolved machine type: ${resolvedMachineType.toString}")
       total = MemorySize.fromMb(resolvedMachineType.getMemoryMb.toDouble)
     } yield {
-      // total - dataproc allocated - welder allocated
-      val dataprocAllocated = config.dataprocConfig.dataprocReservedMemory.map(_.bytes).getOrElse(0L)
-      val welderAllocated = config.welderConfig.welderReservedMemory.map(_.bytes).getOrElse(0L)
-      val result = MemorySize(total.bytes - dataprocAllocated - welderAllocated)
+      // For dataproc, we don't need much memory for Jupyter.
+      val jupyterAllocated =
+        config.dataprocConfig.jupyterReservedMemory.map(_.bytes).getOrElse(MemorySize.fromGb(1.5).bytes)
```

Comment:
Same comment here and below about not hardcoding the memory allocated

Reply:
Same reply, their correct values will be in reference. The "OrElse" part should never be touched.